Frontend/ui-driver/src/Flow.purs

```diff
@@ -180,6 +180,8 @@ baseAppFlow baseFlow event = do
     initialFlow :: FlowBT String Unit
     initialFlow = do
       let regToken = getValueToLocalStore REGISTERATION_TOKEN
+      (API.GetCityRes apiResp) <- Remote.getMerchantOperatingCityListBT ""
```

Comment:
@MercyQueen  If this API fails here, It will do go back and App will not open.
I don't think this API is inside critical PATH, Please confirm this and if it is not critical PATH please add nonBT API and let the flow RUN